README.md

```
@@ -1,33 +1,34 @@
-![Tests Passing](https://github.com/NicolaWealth/rate_limit/actions/workflows/auto_test_main_badge.yml/badge.svg)
+# rate_limit
+
+![Tests Passing](https://github.com/NicolaWealth/rate_limit/actions/workflows/test.yml/badge.svg)
 ![Code Cov](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fgithub.com%2Fnicolawealth%2Frate_limit%2Fraw%2Fmain%2Fcodecov/badge.json&query=%24.message&label=Code%20Coverage&color=%24.color)
 
-# Introduction
-A lightweight utility for rate-limiting function calls in JavaScript/TypeScript. It helps control execution frequency and manage timing for synchronous and asynchronous operations.
+Lightweight utility for rate-limiting function calls in JavaScript/TypeScript. It helps control execution frequency and manage timing for synchronous and asynchronous operations.
 
-# Installation
+## Installation
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)